Mobile Buy SDK/Mobile Buy SDK/Models/BUYCheckout.h

```diff
@@ -225,7 +220,7 @@
 @property (nonatomic, copy, readonly) NSString *sourceName;
 
 /**
- *  The unique identifier for the source: the channelId
+ *  The unique identifier for the source: the appId
  */
 @property (nonatomic, copy, readonly) NSString *sourceIdentifier;
```

Comment:
It's no longer necessary to send when you create/update checkouts, it will be inferred from the api client making the request.